src/main/java/slimeknights/tconstruct/library/client/RenderUtil.java.rej

```diff
@@ -0,0 +1,16 @@
+diff a/src/main/java/slimeknights/tconstruct/library/client/RenderUtil.java b/src/main/java/slimeknights/tconstruct/library/client/RenderUtil.java	(rejected hunks)
```

Comment:
Yeah, something was a little odd which unintended formatting, so I just rejected a couple hunks from one file. Apparently it left a note behind saying what I did...